lib/system_win32.c

```diff
@@ -55,12 +55,7 @@ CURLcode Curl_win32_init(long flags)
     WSADATA wsaData;
     int res;
 
-#if defined(ENABLE_IPV6) && (USE_WINSOCK < 2)
-#error IPV6_requires_winsock2
-#endif
-
-    wVersionRequested = MAKEWORD(USE_WINSOCK, USE_WINSOCK);
-
+    wVersionRequested = MAKEWORD(2, 0);
```

Comment:
I thought 2.0 was more a theoretical thing and never left draft status. All operating systems since Windows 95 OSR2 support 2.2 according to https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-wsastartup, and that's what's requested in virtually all code I've seen so far.

Reply:
[According to Wikipedia](https://en.wikipedia.org/wiki/Winsock#Microsoft_implementations) "Version 2.1 of Winsock was supplied in an add-on package for Windows 95." - so maybe 2.1 is the right version to use if we still want to support Windows 95?

Reply:
Yeah, that was the Winsock2 update for the original version of Windows 95. I'm very sure my OSR2 (Windows 95B) from 1996 supported Winsock 2.2 as that's what I used for my first Winsock-based program :-) Probably doesn't make much sense to support anything but the latest version (which would be C / OSR 2.5 from 1997) of Windows 95?

Reply:
What about Windows CE? "Windows CE .NET 4.1 and later supports Winsock 2.2" is stated [here](https://docs.microsoft.com/en-us/previous-versions/windows/embedded/ms899586(v=msdn.10)). Do we need to support any holder version?

Reply:
Great find, I hadn't thought about Windows CE at all! It's at least good to document as you already did 👍